module/bdev/raid/raid1.c

```diff
@@ -465,8 +508,22 @@ static void
 raid1_ioch_destroy(void *io_device, void *ctx_buf)
 {
 	struct raid1_io_channel *r1ch = ctx_buf;
+	struct raid1_info *r1info = io_device;
+	struct raid_bdev *raid_bdev = r1info->raid_bdev;
+	uint8_t i;
 
 	free(r1ch->read_blocks_outstanding);
```

Comment:
Just curious. Do you know what read_blocks_outstanding means?

Reply:
```
/* Array of per-base_bdev counters of outstanding read blocks on this channel */
uint64_t *read_blocks_outstanding;
```
It is a counter used to balance the amount of data read between the different base bdevs.